common/djangoapps/third_party_auth/models.py

```diff
@@ -882,10 +882,33 @@ class Meta(object):
 
 class ProviderApiPermissions(models.Model):
     """
-    This model links OAuth2 client with provider Id.
+    DEPRECATED: This model links OAuth2 client with provider Id.
 
     It gives permission for a OAuth2 client to access the information under certain IdPs.
 
+    This model is no longer in use.
+
+    TODO: BOM-1270: This model and table will be removed post-Juniper
+    so Open edX instances can migrate data if necessary.
+    To migrate, use ApplicationAccess and add a ``filter`` of the form
+    ``content_org:<ORG NAME>`` (e.g. content_org:edx), for each record
+    in this model's table.
+
+    This model was used by ThirdPartyAuthProviderApiPermission for authorization.
+    The permission class was removed as part of the effort to move from
+    DOP(django-oauth-provider) to DOT(django-oauth-toolkit). TPA_PERMISSIONS class works as
+    a replacement for ThirdPartyAuthProviderApiPermission. To switch over, Open edX instances
+    will need to move their OAuth2 application from DOP to DOT, create the Application Access with the filter,
+    and also create a record in Restricted Application for the DOT application.
+
+    # pylint: disable==line-too-long
+    The first steps regarding DOP will need to review to use SQL from the DB,
+    because admin will be gone. Please look at:
```

Comment:
1. Replace with
```
For additional help, see:
- https://github.com/edx/edx-platform/blob/372d2e927c9eec2a5342ab120f84cc290578aa90/openedx/core/djangoapps/oauth_dispatch/docs/how_tos/testing_manually.rst
```
2. You can move a note above that the table `third_party_auth_providerapipermissions` contains the data that needs to be migrated.